src/test/java/marquez/api/JobIntegrationTest.java

```diff
@@ -255,12 +255,12 @@ private void evaluateResponse(Response res, Job inputJob) {
   }
 
   private Response createJobOnNamespace(String namespace, Job job) {
-    CreateJobRequest createJobRequest =
-        new CreateJobRequest(
-            job.getLocation(),
-            job.getDescription(),
+    JobRequest createJobRequest =
+        new JobRequest(
             job.getInputDataSetUrns(),
-            job.getOutputDataSetUrns());
```

Comment:
I guess the plan would be to later normalize  '..DataSet..' to '..Dataset..' everywhere, also for consistency

Reply:
Yep, exactly! Mind opening an issue to capture the change?